input/pagecontent/index.md

```diff
@@ -50,13 +50,34 @@ The RESTful interaction requirements for AU eRequesting actors, such as read and
 
 The diagram below illustrates the interactions between the various actors:
 
+<figure>
 <div> 
   <img src="au-erequesting-actor-interactions.svg" alt="FHIR RESTful read interactions" style="width:80%"/>
 </div>
+<figcaption>Figure 1: FHIR RESTful interactions</figcaption>
```

Comment:
Figures aren't in fig captions.

Needs to be restored to be `*Figure 1: FHIR RESTful interactions*`